<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSlider_process">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>500</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>play/pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_record">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>record</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_volume">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>22</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_volume">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_thetime">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>thetime</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_media">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>260</width>
      <height>280</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stoprecording">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>stop_r</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_record">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_record">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_selection">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>54</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>54</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>54</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>54</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choose">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recordtime">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>390</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>recordtime</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
